filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -278,23 +278,23 @@ VulkanDriver::VulkanDriver(VulkanPlatform* platform,
 
     // For diagnostic purposes, print useful information about available depth formats.
     // Note that Vulkan is more constrained than OpenGL ES 3.1 in this area.
-#if VK_ENABLE_VALIDATION
-    const VkFormatFeatureFlags required = VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT |
-            VK_FORMAT_FEATURE_SAMPLED_IMAGE_BIT;
-    utils::slog.i << "Sampleable depth formats: ";
-    for (VkFormat format = (VkFormat) 1;;) {
-        VkFormatProperties props;
-        vkGetPhysicalDeviceFormatProperties(mContext.physicalDevice, format, &props);
-        if ((props.optimalTilingFeatures & required) == required) {
-            utils::slog.i << format << " ";
-        }
-        if (format == VK_FORMAT_ASTC_12x12_SRGB_BLOCK) {
-            utils::slog.i << utils::io::endl;
-            break;
+    if (VK_ENABLE_VALIDATION && FILAMENT_VULKAN_VERBOSE) {
```

Comment:
I started to use `if constexpr` for these cases.

Reply:
Nice, done.